include/fmt/std.h

```diff
@@ -214,10 +214,41 @@ struct formatter<
     std::size_t size = 0;
     std::unique_ptr<char, decltype(&std::free)> demangled_name_ptr(
         abi::__cxa_demangle(ti.name(), nullptr, &size, &status), &std::free);
-    out = detail::write_bytes(
-        out,
-        string_view(demangled_name_ptr ? demangled_name_ptr.get() : ti.name()),
-        spec);
+
+    string_view demangled_name_view;
+    if (demangled_name_ptr) {
+      demangled_name_view = string_view(demangled_name_ptr.get());
```

Comment:
I don't think you need to explicitly invoke the `string_view` ctor here.

Reply:
``string_view`` does not contain assignment operators/functions for ``char *`` and ``{ char *, size_t }`` pair.
See: https://en.cppreference.com/w/cpp/string/basic_string_view